Source/Core/DolphinQt/Settings.cpp

```diff
@@ -335,11 +335,11 @@ void Settings::RemovePath(const QString& qpath)
   std::string path = qpath.toStdString();
   std::vector<std::string> paths = Config::GetIsoPaths();
 
-  auto new_end = std::remove(paths.begin(), paths.end(), path);
-  if (new_end == paths.end())
+  const auto remove_result = std::ranges::remove(paths, path);
```

Comment:
```
if (std::erase(paths, path) == 0)
  return;
```

?

Reply:
Reviewing this code again, the early return here is awfully paranoid. Not my concern, though.